### ISSUE TITLE
[BUG] - Missing serving size when printing recipe

### ISSUE THREAD

**Issue by thomaspijper** [mealie-recipes/mealie#3788]
### First Check

- [X] This is not a feature request.
- [X] I added a very descriptive title to this issue (title field is above this).
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [X] This issue can be replicated on the demo site (https://demo.mealie.io/).

### What is the issue you are experiencing?

When a recipe is printed (to paper or PDF -- doesn't matter), the serving size is missing.

### Steps to Reproduce

1) Open a recipe
2) Click the blue circle with the three dots that is on the bottom right of the recipe image.
3) Choose "print"
4) Both the preview and the print result do not show the serving size.

The "print preferences" view also does not show the serving size

### Please provide relevant logs

No relevant logs

### Mealie Version

Docker image ghcr.io/mealie-recipes/mealie, v1.9.0

### Deployment

Docker (Linux)

**Comment by boc-the-git** [comment]
Whilst "bug" is probably a grey area and this could be argued as a feature request, happy for someone to add this!

**Comment by thomaspijper-philips** [comment]
IMO, the serving size is a fundamental piece of information in a recipe, which is why I labelled it a bug (hereby assuming it is meant to be shown but doesn't render correctly). But checking `RecipePrintView.vue` I see that it is not implemented. Guess that does make it closer to a feature request.

Would be grateful if someone could add this. Typescript is not my forte.

**Comment by thomaspijper** [comment]
IMO, the serving size is a fundamental piece of information in a recipe, which is why I labelled it a bug (hereby assuming it is meant to be shown but doesn't render correctly). But checking `RecipePrintView.vue `I see that it is not implemented. Guess that does make it closer to a feature request.

Would be grateful if someone could add this. Typescript is not my forte.

Edit: I made an effort. See below PR.
